src/components/TabBar.tsx

```diff
@@ -1,30 +1,66 @@
 'use client';
 import {Link} from '@/navigation';
 import {useUserStore} from '@/store/userStore';
-import {Home, ListTodo, LogIn, Settings} from 'lucide-react';
+import {usePathname} from 'next/navigation';
 import {FC} from 'react';
 
+import {Home, Library, ListTodo, Settings, Users} from 'lucide-react';
+
 export const TabBar: FC = () => {
+  const pathname = usePathname();
   const {user} = useUserStore();
 
+  const colorText = (path: string) => {
+    if (pathname.includes(path)) {
+      return 'text-green';
+    } else {
+      return 'text-black';
+    }
+  };
+
   return (
-    <div className="fixed bottom-0 flex h-[40px] w-full items-center justify-between border-t-2 border-black bg-yellow px-14">
-      <span>
-        <Link href="/">
-          <Home size={24} />
-        </Link>
-      </span>
-      <span>
-        <Link href={!user ? '/user' : '/user/profile'}>
-          <LogIn size={24} />
-        </Link>
-      </span>
-      <span>
-        <Settings size={24} />
-      </span>
-      <span>
-        <ListTodo size={24} />
-      </span>
-    </div>
+    <nav className="fixed bottom-0 h-[40px] border-t-2 border-black bg-yellow w-full">
```

Comment:
The tab bar ends up behind this scroll thing on iphones. We we need a little padding at the bottom. 32px is usually the standard to clear that scroller 

![IMG_3016](https://github.com/user-attachments/assets/10df2eda-7b0c-4535-bebb-119ce6fd06d0)